coralproject/talk
# Fix/corl 2468 ban modal fixes

## What does this PR do?
In the UserBanModal:
- displays "Specific Sites" by default if the user in question has any single site bans
- only displays "No Sites" option if user is currently banned across all sites (and viewer is admin)

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## How do I test this PR?
- open the user ban modal for a user with a single site ban and no blanket ban, and observe the the "Specific Sites" tab is selected by default, showing the individual sites on which the user is banned
- Open the UserBanModal as a non-admin user and observe that the "No Sites" option is not present (as currently only admins can remove bans)
 
## How do we deploy this PR?
No special considerations should be required.
